install/roles/server/tasks/uninstall_podman_containers.yml

```diff
@@ -6,7 +6,7 @@
   register: podman_postgres_ps_raw
   ignore_errors: yes
 
-- name: Set variable indicating whether QPC server podman container found
+- name: Set variable indicating whether quipucords server PostgreSQL podman container found
```

Comment:
change to `Set variable indicating whether the Quipucords server PostgreSQL podman container is present`